cmd/cluster/sre_operators/listCmd.go

```diff
@@ -0,0 +1,477 @@
+package sre_operators
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"regexp"
+	"runtime"
+	"sort"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/google/go-github/v63/github"
+	"github.com/openshift/osdctl/pkg/printer"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/xanzy/go-gitlab"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/cmd/util"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+type sreOperatorsListOptions struct {
+	short     bool
+	outdated  bool
+	noHeaders bool
+	operator  string
+
+	genericclioptions.IOStreams
+	kubeCli client.Client
+}
+
+type sreOperator struct {
+	Name     string
+	Current  string
+	Expected string
+	Status   string
+	Channel  string
+}
+
+const (
+	sreOperatorsListExample = `
+	# List SRE operators
+	$ osdctl cluster sre-operators list
+	
+	# List SRE operators without fetching the latest version for faster output
+	$ osdctl cluster sre-operators list --short
+	
+	# List only SRE operators that are running outdated versions
+	$ osdctl cluster sre-operators list --outdated
+	
+	# List a specific SRE operator
+	$ osdctl cluster sre-operators list --operator='OPERATOR_NAME'
+	`
+	sreOperatorsListDescription = `
+	Lists the current version, channel, and status of SRE operators running in the current 
+	cluster context, and by default fetches the latest version from the operators' repositories.
+	
+	A GitLab access token is required to fetch the latest version of the operators, and can be 
+	set in the '.config/osdctl' file as 'gitlab_access: <TOKEN>', or using the 'osdctl setup' command.
```

Comment:
If we are providing a command to configure the credential programmatically, we shouldn't encourage people to do it manually.

Reply:
I have changed the message, thank you for the suggestion!